dipy/io/stateful_tractogram.py

```diff
@@ -110,11 +131,45 @@ def __init__(self, streamlines, reference, space,
             raise ValueError('Space MUST be from Space enum, e.g Space.VOX')
         self._space = space
 
+        if origin_at_corner in Origin:
+            origin_at_corner = origin_at_corner.value
+
         if not isinstance(origin_at_corner, bool):
-            raise TypeError('origin_at_corner MUST be a boolean')
+            raise TypeError('origin_at_corner MUST be a boolean or from '
+                            'Origin enum, e.g Origin.NIFTI')
         self._origin_at_corner = origin_at_corner
         logger.debug(self)
 
+    @staticmethod
+    def from_sft(streamlines, sft,
+                 data_per_point=None,
+                 data_per_streamline=None):
+        """ Re-create a strict, state-aware, robust tractogram from another
+
+        Parameters
+        ----------
+        streamlines : list or ArraySequence
+            Streamlines of the tractogram
+        sft : StatefulTractgram,
+            The other StatefulTractgram to copy the space_attribute AND
+            state from.
+        data_per_point : dict
```

Comment:
```suggestion
        data_per_point : dict, optional
```

Also: would be good to explain what the default behavior would be (presumably inherits from the source SFT?)

Reply:
Oh - I see. If it's not specified it's set to None. Of course.